'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _defineProperty2 = require('babel-runtime/helpers/defineProperty');

var _defineProperty3 = _interopRequireDefault(_defineProperty2);

var _conferenceCallErrors;

var _conferenceCallErrors2 = require('ringcentral-integration/modules/ConferenceCall/conferenceCallErrors');

var _conferenceCallErrors3 = _interopRequireDefault(_conferenceCallErrors2);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = (_conferenceCallErrors = {}, (0, _defineProperty3.default)(_conferenceCallErrors, _conferenceCallErrors3.default.bringInFailed, 'Failed to merge the calls due to unexpected errors. Please try again later.'), (0, _defineProperty3.default)(_conferenceCallErrors, _conferenceCallErrors3.default.makeConferenceFailed, 'Failed to merge the calls due to unexpected errors. Please try again later.'), (0, _defineProperty3.default)(_conferenceCallErrors, _conferenceCallErrors3.default.terminateConferenceFailed, 'Failed to hangup the conference due to unexpected errors. Please try again later.'), (0, _defineProperty3.default)(_conferenceCallErrors, _conferenceCallErrors3.default.removeFromConferenceFailed, 'Failed to remove the participant due to unexpected errors. Please try again later.'), (0, _defineProperty3.default)(_conferenceCallErrors, _conferenceCallErrors3.default.callIsRecording, 'Call recording in progress. Please stop recording and try again.'), _conferenceCallErrors);
//# sourceMappingURL=en-US.js.map
